polyfill/test/ecmascript.mjs

```diff
@@ -13,6 +13,10 @@ import * as ES from '../lib/ecmascript.mjs';
 import { GetSlot, TIMEZONE_ID } from '../lib/slots.mjs';
 import { TimeZone } from '../lib/timezone.mjs';
 
+// eslint-disable-next-line max-len
+// curl -s https://raw.githubusercontent.com/unicode-org/cldr-json/main/cldr-json/cldr-bcp47/bcp47/timezone.json > cldr-timezone.json
+import cldrTimeZoneJson from './cldr-timezone.json' assert { type: 'json' };
```

Comment:
I wonder if it's better to parse manually, since `assert` precludes the possibility of running this in older implementations and has been [renamed to `with`](https://github.com/tc39/proposal-import-attributes#history) anyway.
```suggestion
import { readFileSync } from 'fs';
const cldrTimeZonePath = new URL('./cldr-timezone.json', import.meta.url);
const cldrTimeZoneJson = JSON.parse(readFileSync(cldrTimeZonePath));
```

Reply:
Good idea. I'd never used those assert things anyways, so glad to see they're gone. Good riddance.